source/include/beam6d.f90

```diff
@@ -22,13 +22,12 @@
   omoidpsv(j)=c1e3*((one-mtc(j))*oidpsv(j))
   yv(1,j)=((track6d(2,j)*c1e3+clobeam(4,imbb(i)))-beamoff(4,imbb(i)))*moidpsv(j)
   yv(2,j)=((track6d(4,j)*c1e3+clobeam(5,imbb(i)))-beamoff(5,imbb(i)))*moidpsv(j)
-
   !TODO check for ions?
   
   ejfv(j)=(dpsv(j)*e0f+e0f)*(nucm(j)/nucm0) 
   ejv(j)=sqrt(ejfv(j)**2+nucm(j)**2)
   rvv(j)=(ejv(j)*e0f)/(e0*ejfv(j))
-  
+  dpsv1(j)=(dpsv(j)*c1e3)*oidpsv(j)
```

Comment:
Why? I thought this was fixed in the other pull request on energy variable

Reply:
I think this is some fail linked to git and my merge of master into the branch because the line is there in master. I had some conflicts and maybe this caused some confusion for git.

Reply:
it is a bit fishy though, I would like to understand more...

BTW with a grep I have just found
```
dpsv1(j)=(dpsv(j)*c1e3)*moidpsv(j)
```
in track_thick that should be wrong to my understanding. Can you confirm?

Reply:
Yeah I can confirm. Must have missed it.